src/app/common/directives.js

```diff
@@ -1,40 +1,38 @@
-(function () {
+(function() {
   'use strict';
 
-  function isActive ($location) {
+  function isActive($location) {
     return {
       link: function(scope, elm, attr) {
         scope.$on('$routeChangeSuccess', function() {
           var href = '/' + attr.href.split('/')[2];
+
           if ($location.path() === href) {
             elm.addClass('nav-active');
-          }
-          else {
+          } else {
             elm.removeClass('nav-active');
           }
         });
       }
     };
   }
 
-  function isTransparent ($location) {
+  function isTransparent($location) {
     return {
       link: function(scope, elm) {
         scope.$on('$routeChangeSuccess', function() {
           if ($location.path() === '/') {
             elm.addClass('transparent');
-          }
-          else {
+          } else {
             elm.removeClass('transparent');
           }
         });
       }
     };
   }
 
-
-  angular.module('Directives', [])
+  angular
```

Comment:
not sure this is a best practice, it doesnt hurt anything, but it can become unclear.  just a person preference and the preference of my mentor